src/render/mesh/model.rs

```diff
@@ -8,81 +8,91 @@ use super::quad::QuadUV;
 
 #[derive(Debug, Clone)]
 pub struct Model {
-    pub vertices: Vec<Vertex>
+    pub vertices: Vec<Vertex>,
 }
 
 const HALF_SQRT_2: f32 = SQRT_2 * 0.5;
 const QUARTER_SQRT_2: f32 = SQRT_2 * 0.25;
 const QUAD_INDICES: [usize; 6] = [0, 1, 2, 0, 2, 3];
-const PLANT_INDICES: [usize; 12] = [
-    0, 1, 2, 0, 2, 3,
-    4, 5, 6, 4, 6, 7,
-];
+const PLANT_INDICES: [usize; 12] = [0, 1, 2, 0, 2, 3, 4, 5, 6, 4, 6, 7];
 
 const DEFAULT_PLANT_VERT_POSITIONS: [Vec3; 8] = [
     Vec3::new(-QUARTER_SQRT_2, -0.5, -QUARTER_SQRT_2),
-    Vec3::new( QUARTER_SQRT_2, -0.5,  QUARTER_SQRT_2),
-    Vec3::new( QUARTER_SQRT_2,  0.5,  QUARTER_SQRT_2),
-    Vec3::new(-QUARTER_SQRT_2,  0.5, -QUARTER_SQRT_2),
-
-    Vec3::new( QUARTER_SQRT_2, -0.5, -QUARTER_SQRT_2),
-    Vec3::new(-QUARTER_SQRT_2, -0.5,  QUARTER_SQRT_2),
-    Vec3::new(-QUARTER_SQRT_2,  0.5,  QUARTER_SQRT_2),
-    Vec3::new( QUARTER_SQRT_2,  0.5, -QUARTER_SQRT_2),
+    Vec3::new(QUARTER_SQRT_2, -0.5, QUARTER_SQRT_2),
+    Vec3::new(QUARTER_SQRT_2, 0.5, QUARTER_SQRT_2),
+    Vec3::new(-QUARTER_SQRT_2, 0.5, -QUARTER_SQRT_2),
+    Vec3::new(QUARTER_SQRT_2, -0.5, -QUARTER_SQRT_2),
+    Vec3::new(-QUARTER_SQRT_2, -0.5, QUARTER_SQRT_2),
+    Vec3::new(-QUARTER_SQRT_2, 0.5, QUARTER_SQRT_2),
+    Vec3::new(QUARTER_SQRT_2, 0.5, -QUARTER_SQRT_2),
```

Comment:
:C

Reply:
I think this one is perfectly fine, since the whole purpose of the `vec!` macro is to serve this exact purpose.